shared-bindings/busio/SPI.c

```diff
@@ -74,7 +74,8 @@
 //|
 //|         :param ~microcontroller.Pin clock: the pin to use for the clock.
 //|         :param ~microcontroller.Pin MOSI: the Main Out Selected In pin.
-//|         :param ~microcontroller.Pin MISO: the Main In Selected Out pin."""
+//|         :param ~microcontroller.Pin MISO: the Main In Selected Out pin.
+//|         :param bool half_duplex: True when MOSI is used for bidirectional data. False when SPI is full-duplex or simplex."""
```

Comment:
Note that we should have a `half_duplex: bool = False` in the function signature.

Reply:
Where is that?

Reply:
Address by #6003.

Reply:
Never mind, looks like dhalbert is on it